scripts/infra

```diff
@@ -27,7 +27,9 @@ function update_batch_definitions() {
     docker compose build
 
     ENVIRONMENT="${ENVIRONMENT:-staging}"
-    BATCH_ANALYSIS_JOB_NAME_REVISION=$(docker compose run --rm \
+    BATCH_ANALYSIS_JOB_NAME_REVISION=$(AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} \
```

Comment:
There might be a better way to pass them in, but we don't use a dotenv file as the env file in `docker-compose`, so this is the most straightforward way to do it I think. Please suggest if there is a more ideal solution!